packages/js/src/helpers/stringHelpers.js

```diff
@@ -31,15 +31,16 @@ export function stripHTML( string ) {
  *
  * @param {string} text   The text to add links to. Make sure it contains <a> and </a> tags surrounding the link part.
  * @param {string} linkTo The target URL for the link (href).
+ * @param {string} id     The id to attach to the link.
  *
  * @returns {WPElement} A Fragment with the text and a link.
  */
-export function addLinkToString( text, linkTo ) {
+export function addLinkToString( text, linkTo, id = "" ) {
 	return createInterpolateElement(
 		text,
 		{
 			// eslint-disable-next-line jsx-a11y/anchor-has-content
-			a: <a href={ linkTo } target="_blank" rel="noopener noreferrer" />,
+			a: <a id={ id } href={ linkTo } target="_blank" rel="noopener noreferrer" />,
```

Comment:
just checking my knowledge: what happens if the `id` param is left empty? I suppose the element is rendered without an `id` at all, right? not an empty id

Reply:
Yes without an id as far as I know.

Reply:
This actually renders with an empty id. Not sure how damaging that is, but the way to keep attributes out of html from jsx is setting them to `null`.

This is with an empty string:
![image](https://user-images.githubusercontent.com/26220788/143545070-e0a637bb-5ad3-447e-8bdd-f4487bbfd789.png)